neo/SmartContract/SmartContractEvent.py

```diff
@@ -231,11 +229,18 @@ def __init__(self, event_type, event_payload, contract_hash, block_number, tx_ha
                     self.amount = int(BigInteger.FromBytes(event_payload[3])) if isinstance(event_payload[3], bytes) else int(event_payload[3])
                     self.is_standard_notify = True
 
-                elif plen == 3 and self.notify_type == NotifyType.REFUND:
+                elif self.notify_type == NotifyType.REFUND and plen >= 3:  # Might have more arguments
```

Comment:
Saw some contracts that have 4 arguments, using this signature:

    OnRefund = RegisterAction('refund', 'addr_to', 'amount', 'asset')

Reply:
Interesting, I wonder what the 4th argument is?

Reply:
whether it's gas or neo?